docs/src/components/index.md

```diff
@@ -18,41 +18,39 @@ This component automatically provides the search state to all its children.
 Basic usage:
 
 ```html
-<ais-index index-name="your_indexName"
-           app-id="YourAppID"
-           api-key="YourSearchAPIKey"
->
-  <!-- Add your InstantSearch components here. -->
-</ais-index>
-```
+<template>
+  <ais-index index-name="your_indexName"
+            :searchClient="searchClient"
+  >
+    <!-- Add your InstantSearch components here. -->
+  </ais-index>
+</template>
 
-Provide search query parameters:
+<!-- You need to instantiate the search client in your script -->
+<script>
+import algoliasearch from 'algoliasearch/lite';
 
-```html
-<ais-index index-name="your_indexName"
-           app-id="YourAppID"
-           api-key="YourSearchAPIKey"
-           :query-parameters="{
-             distinct: true,
-             attributeForDistinct: 'product_id'
-           }"
->
-  <!-- Add your InstantSearch components here. -->
-</ais-index>
+export default {
+  data() {
+    return {
+      searchClient: algoliasearch(
+        'latency',
+        '3d9875e51fbd20c7754e65422f7ce5e1'
+      ),
+    };
+  },
+};
+</script>
 ```
 
-
 ## Props
 
-| Name             | Type    | Default | Description                                                                                                                                        |
+  | Name             | Type    | Default | Description                                                                                                                                        |
```

Comment:
could this table be reformatted to have the same info and formatting as the other ones?